shared/domains/chelonia/chelonia.js

```diff
@@ -108,6 +209,15 @@ export default (sbp('sbp/selectors/register', {
       return stack
     }
   },
+  'chelonia/with-env': async function (contractID: string, env: Object, sbpInvocation: Array<*>) {
```

Comment:
- For consistence with everything else, the selector should be renamed to `'chelonia/withEnv'`
- Semantically, I think it might make more sense to either make the first parameter the env, or — and probably even better — to make the first parameter be an object instead that contains these other parameters

As an aside:

> make the first parameter be an object instead that contains these other parameters

I think this is something we should've done throughout our codebase anywhere this is more than 1 parameter. Named parameters make the code much easier to read. I think Objective-C got this right.

Reply:
I'll update this!

Reply:
It sounds like we might not need the `contractID` parameter because of: https://github.com/okTurtles/group-income/pull/1257#pullrequestreview-1155587120